backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/shippingnotification/ShippingNotificationFromShipmentScheduleProducer.java

```diff
@@ -50,12 +50,6 @@ public ProcessPreconditionsResolution checkCanCreateShippingNotification(@NonNul
 			return ProcessPreconditionsResolution.rejectWithInternalReason("only completed orders");
 		}
 
-		final YearAndCalendarId harvestingYearId = extractHarvestingYearId(salesOrder).orElse(null);
-		if (harvestingYearId == null)
-		{
-			return ProcessPreconditionsResolution.reject(MSG_M_Shipment_Notification_NoHarvestingYear);
```

Comment:
shall we also remove MSG_M_Shipment_Notification_NoHarvestingYear from code and from AD_Message?

Reply:
yes. Removed it